xmlWorkbook.go

```diff
@@ -155,6 +219,25 @@ type xlsxSheet struct {
 	State   string `xml:"state,attr,omitempty"`
 }
 
+// tmpXlsxSheet is struct for custom marshal xlsxSheet
+type tmpXlsxSheet struct {
```

Comment:
You used anonymous structs inside the function in other places but not here. It would be good to stay consistent.